doc/rados/operations/placement-groups.rst

```diff
@@ -145,21 +145,21 @@ the number of those respective device types.
 
 The autoscaler uses the `bulk` flag to determine which pool
 should start out with a full complements of PGs and only
```

Comment:
Thing about doc PRs is that you go in to change one thing and find more.  I just noticed that "complements" here should be "complement", mind adding that to your PR?

Reply:
sure thing